aiohttp/client.py

```diff
@@ -428,7 +428,9 @@ async def _request(
         ]
 
         for trace in traces:
-            await trace.send_request_start(method, url.update_query(params), headers)
+            await trace.send_request_start(
+                method, url.update_query(params or {}), headers
```

Comment:
Does it make more sense to just do `params = params or {}` further up, rather than repeated the same code 4 times?

Reply:
done